src/components/treePage/treeInfo.tsx

```diff
@@ -235,3 +240,13 @@ const TreePageShareButton: React.FC<TreePageShareButtonProps> = ({
     />
   );
 };
+
+interface TreeUploadProps {
+  readonly siteData: SiteProps;
+}
+
+const TreePageUploadSiteImageButton: React.FC<TreeUploadProps> = ({
+  siteData,
+}) => {
+  return <UploadSiteImageButton siteEntryId={siteData.entries[0].id} />;
+};
```

Comment:
❓ Just curious, how come we have this extra component rather than directly using `UploadSiteImageButton`? Was there some weird bug that didn't show the button without this?